slf4j/src/test/scala/org/typelevel/log4cats/slf4j/LoggerNameTest.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.slf4j
+
+import munit.FunSuite
+
+class LoggerNameTest extends FunSuite {
+  class FooA {
+    val name = LoggerName.name
+  }
+
+  object FooB {
+    val name = LoggerName.name
+  }
+
+  test("names") {
+    val name1 = new FooA().name
+    val name2 = FooB.name
+
+    assertEquals(name1.value, "org.typelevel.log4cats.slf4j.LoggerNameTests.FooA")
+    assertEquals(name2.value, "org.typelevel.log4cats.slf4j.LoggerNameTests.FooB")
```

Comment:
This will fix tests, I guess:
```suggestion
    assertEquals(name1.value, "org.typelevel.log4cats.slf4j.LoggerNameTest.FooA")
    assertEquals(name2.value, "org.typelevel.log4cats.slf4j.LoggerNameTest.FooB")
```